pkg/localnet/localnet.go

```diff
@@ -185,22 +184,41 @@ func LocalNetworkHealth(
 	if err != nil {
 		return pChainBootstrapped, false, err
 	}
+	clusters, err := GetRunningLocalClustersConnectedToLocalNetwork(app)
+	if err != nil {
+		return pChainBootstrapped, false, err
+	}
 	for _, blockchain := range blockchains {
 		hasValidators, err := LocalNetworkHasValidatorsForSubnet(app, blockchain.SubnetID)
 		if err != nil {
 			return pChainBootstrapped, false, err
 		}
 		if !hasValidators {
-			printFunc(logging.Red.Wrap("local network has no validators for subnet %s. l1 check is not implemented yet"), blockchain.SubnetID)
-			printFunc("")
-			return pChainBootstrapped, false, err
-		}
-		blockchainBootstrapped, err := IsLocalNetworkBlockchainBootstrapped(app, blockchain.ID.String(), blockchain.SubnetID)
-		if err != nil {
-			return pChainBootstrapped, false, err
-		}
-		if !blockchainBootstrapped {
-			return pChainBootstrapped, false, nil
+			blockchainBootstrappedOnSomeCluster := false
+			for _, clusterName := range clusters {
+				if hasValidators, err := LocalClusterHasValidatorsForSubnet(app, clusterName, blockchain.SubnetID); err != nil {
+					return pChainBootstrapped, false, err
+				} else if !hasValidators {
+					continue
+				}
+				blockchainBootstrapped, err := IsLocalClusterBlockchainBootstrapped(app, clusterName, blockchain.ID.String(), blockchain.SubnetID)
```

Comment:
why not just assign blockchainBootstrappedOnSomeCluster here

Reply:
we iterate over all the clusters , the last one can be false and overwrite some previous true value. I believe it is cleaner this way that to assign to blockchainBootstrappedOnSomeCluster, check the value and continue if true

Reply:
ok will implement the version with break